client/analytics/report/products/index.js

```diff
@@ -75,6 +77,16 @@ class ProductsReport extends Component {
 				compareObject === 'products' ? 'product' : 'variation';
 		}
 
+		filters[ 0 ].filters[ 2 ].settings.onClick = () => {
+			addCesSurveyTrack(
```

Comment:
Can you extract this `onClick` handler into a new file so it's de-duplicated?

Reply:
Extracted to action in https://github.com/woocommerce/woocommerce-admin/pull/5649/commits/8204aa31cf41f819b56943977a03b9519a345d29